README.md

```diff
@@ -168,16 +168,18 @@ Here is an overview of all the currently supported files and formats.
 | -------------- | ------ |-| ----------------------------- | ------ |
 | (.)somesy.toml | ✓      | | pyproject.toml _(poetry)_     | ✓      |
 | pyproject.toml | ✓      | | pyproject.toml _(setuptools)_ | ✓(1.)  |
-| package.json   | ✓      | | package.json                  | ✓(2.)  |
-| Project.toml   | ✓      | | Project.toml                  | ✓      |
+| package.json   | ✓      | | package.json _(JavaScript)_   | ✓(2.)  |
+| Project.toml   | ✓      | | Project.toml _(Julia)_        | ✓      |
+| fpm.toml       | ✓      | | fpm.toml _(Fortran)_          | ✓(3.)  |
 |                |        | | CITATION.cff                  | ✓      |
-|                |        | | codemeta.json                 | ✓(3.)  |
+|                |        | | codemeta.json                 | ✓(4.)  |
 
 **Notes:**
 
 1. note that `somesy` does not support setuptools *dynamic fields*
 2. `package.json` only supports one author, so `somesy` will pick the *first* listed author
-3. unlike other targets, `somesy` will *re-create* the `codemeta.json` (i.e. do not edit it by hand!)
+3. `fpm.toml` only supports one author and maintainer, so `somesy` will pick the *first* listed author and maintainer
```

Comment:
As currently the format is not specified (https://github.com/fortran-lang/fpm/issues/995), we technically could just concatenate author and maintainer name+emails, like e.g. `Jane Doe <mail1>, John Doe <mail 2>` ;) But maybe that's not a very useful idea... I just mean, technically, there is no restriction whatsoever for `author` and `maintainer` of fpm.toml, and some examples have something like `the XYZ project authors` as the `author` string.

Reply:
I thought of that, too, but I decided to write only one person since the author and maintainer fields have singular key names.